engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/StartPlayingScreen.java

```diff
@@ -70,19 +65,8 @@ public void initialise() {
 
             SimpleUri uri;
             WorldInfo worldInfo;
+            //TODO: if we don't do that here, where do we do it? or does the world not show up in the game manifest?
```

Comment:
This is done as part of `GameManifestProvider#createGameManifest` which is called on the other code paths, I think.

https://github.com/MovingBlocks/Terasology/blob/1d831ead54b3883a149f3671d07819a3d43c4637/engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/GameManifestProvider.java#L96

We can remove this TODO comment and the line below.